lib/include/chunk.h

```diff
@@ -12,7 +12,8 @@
 #define BLOCK_BUFFER_SIZE (CHUNK_SIZE*CHUNK_SIZE*CHUNK_SIZE*BLOCK_SIZE)
 #define BLOCKS_HEADER_SIZE 2
 
-namespace konstructs {
+namespace konstructs 
+{
```

Comment:
I assume this was some automatic refactoring done by your editor/IDE? Please undo these changes because they increases the size of the diff, and they also changes the coding style. Try to look at the surrounding functions coding style and use the same. This will make both the diff and the finalized code easier to read.